daemon/src/model/cfd.rs

```diff
@@ -591,8 +590,8 @@ impl Cfd {
     }
 
     pub fn start_rollover(&self) -> Result<(RolloverParams, Dlc, Duration)> {
-        if !self.can_roll_over() {
-            bail!("Start rollover only allowed when open")
+        if let Err(e) = self.can_roll_over() {
+            bail!(e)
```

Comment:
> ❓ Also, isn't it weird that internally we are using the non-actor-specific version of this API.

Both maker and taker share a set up rules, but the taker additionally checks the time since last rollover (I think the maker should not care about that). I wanted to keep the API consistent, but don't duplicate the code for the shared logic...

Reply:
Your comment made me think again. The logic is actually incorrect at the moment I think. The rollover actor should check if it is possible to roll over, similar to the collaborative settlement actor. The cfd actor should not call `is_rollover_possible` for either party. For the taker the auto-rollover actor will call the taker specific one for decision making!

Reply:
I think I agree!